runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -92,6 +101,39 @@ class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 	virtual bool preMonitorTableSlotRead(J9JavaVM *vm, j9object_t *srcAddress);	
 #endif	
 
+	bool preObjectStoreImpl(J9VMThread *vmThread, J9Object *destObject, fj9object_t *destAddress, J9Object *value, bool isVolatile);
+	bool preObjectStoreImpl(J9VMThread *vmThread, J9Object **destAddress, J9Object *value, bool isVolatile);
+
+	void rememberObjectToRescan(MM_EnvironmentBase *env, J9Object *object);
+
+	MMINLINE bool isSATBBarrierActive(MM_EnvironmentBase* env)
+	{
+		return ((_extensions->configuration->isSnapshotAtTheBeginningBarrierEnabled()) &&
+				(!_extensions->sATBBarrierRememberedSet->isGlobalFragmentIndexPreserved(env)));
+	}
```

Comment:
Extra brackets are really needed when it's not so obvious which operator has priority. I believe that for most people there is no dilemma that -> operator is among the highest. But you can leave it as is, unless you end up changing the code again.